test/machineslib.py

```diff
@@ -347,7 +347,7 @@ def setUp(self):
         # FIXME: report downstream; AppArmor noisily denies some operations, but they are not required for us
         self.allow_journal_messages(r'.* type=1400 .* apparmor="DENIED" operation="capable" profile="\S*libvirtd.* capname="sys_rawio".*')
         # AppArmor doesn't like the non-standard path for our storage pools
-        self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="open" profile="virt-aa-helper" name="%s.*' % self.vm_tmpdir)
+        self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="open" * profile="virt-aa-helper" name="%s.*' % self.vm_tmpdir)
```

Comment:
Test so far fails: https://cockpit-logs.us-east-1.linodeobjects.com/pull-835-20221021-105036-d3235509-ubuntu-stable-bots-3953/log.html#26

Shouldn't it be?
```suggestion
        self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="open" .* profile="virt-aa-helper" name="%s.*' % self.vm_tmpdir)
```